goldilocks/src/lib.rs

```diff
@@ -199,6 +199,12 @@ impl AbstractField for Goldilocks {
     fn generator() -> Self {
         Self::new(7)
     }
+
+    #[inline]
+    fn zero_vec(len: usize) -> Vec<Self> {
+        // SAFETY: repr(transparent) ensures transmutation safety.
+        unsafe { transmute(vec![0u64; len]) }
+    }
```

Comment:
Given how easy the change is, it's probably worth implementing it for the AVX2/AVX512 Goldilocks implementations as well.